### ISSUE TITLE
Cannot find loaded worlds

### ISSUE THREAD

**Issue by stifflered** [RoboMWM/MidnightPortal#1]
Got an error when loading the plugin. Running latest build (489) of PaperSpigot 1.13.2 and the latest versions of WorldGuard and Multiverse.

This happens on server load:

```
[01:46:36] [Server thread/INFO]: [MidnightPortal] Enabling MidnightPortal v1.0
[01:46:36] [Server thread/WARN]: [MidnightPortal] challenge or TheFlow is not a valid/loaded world. 
```

This is my config:
```
portalFrameMaterial: QUARTZ_BLOCK
enabledWorlds:
  challenge:
    TheFlow: 10.0
  TheFlow:
    challenge: 0.1
```

These are the two worlds in Multiverse. They are loaded there and in WorldGuard since I can enter them and modify them as needed.

```
TheFlow:
    ==: MVWorld
    hidden: 'false'
    alias: TheFlow
    color: WHITE
    style: NORMAL
    pvp: 'true'
    scale: '10.0'
    respawnWorld: ''
    allowWeather: 'true'
    difficulty: HARD
    spawning:
      ==: MVSpawnSettings
      animals:
        ==: MVSpawnSubSettings
        spawn: 'false'
        spawnrate: '-1'
        exceptions: []
      monsters:
        ==: MVSpawnSubSettings
        spawn: 'true'
        spawnrate: '-1'
        exceptions: []
    entryfee:
      ==: MVEntryFee
      amount: '0.0'
      currency: '-1'
    hunger: 'true'
    autoHeal: 'true'
    adjustSpawn: 'true'
    portalForm: ALL
    gameMode: SURVIVAL
    keepSpawnInMemory: 'true'
    spawnLocation:
      ==: MVSpawnLocation
      x: 0.0
      y: 71.0
      z: 0.0
      pitch: -1.950006
      yaw: -3.7500005
    autoLoad: 'true'
    bedRespawn: 'true'
    worldBlacklist: []
    environment: NORMAL
    seed: '7414836039109396892'
    generator: 'null'
    playerLimit: '-1'
    allowFlight: 'true'
  challenge:
    ==: MVWorld
    hidden: 'false'
    alias: challenge
    color: WHITE
    style: NORMAL
    pvp: 'false'
    scale: '1.0'
    respawnWorld: ''
    allowWeather: 'true'
    difficulty: NORMAL
    spawning:
      ==: MVSpawnSettings
      animals:
        ==: MVSpawnSubSettings
        spawn: 'true'
        spawnrate: '-1'
        exceptions: []
      monsters:
        ==: MVSpawnSubSettings
        spawn: 'true'
        spawnrate: '-1'
        exceptions: []
    entryfee:
      ==: MVEntryFee
      amount: '0.0'
      currency: '-1'
    hunger: 'true'
    autoHeal: 'true'
    adjustSpawn: 'false'
    portalForm: ALL
    gameMode: SURVIVAL
    keepSpawnInMemory: 'true'
    spawnLocation:
      ==: MVSpawnLocation
      x: -13.526359210147286
      y: 71.0
      z: 41.289882654289045
      pitch: 9.600014
      yaw: -93.15247
    autoLoad: 'true'
    bedRespawn: 'true'
    worldBlacklist: []
    environment: NORMAL
    seed: '2515973426658598677'
    generator: 'null'
    playerLimit: '-1'
    allowFlight: 'true'
```

**Comment by stifflered** [comment]
Randomly, this error popped up when a different player logged in for the first time:

```
[02:02:59] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
[02:03:00] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
[02:03:01] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
[02:03:07] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
[02:03:09] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
[02:03:13] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
[02:03:14] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to MidnightPortal v1.0
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-473]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:314) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:465) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1359) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-473]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-473]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-473]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NullPointerException
	at com.robomwm.midnightportal.listener.PortalLighter.onPlaceFire(PortalLighter.java:42) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.13.2.jar:git-Paper-473]
	... 17 more
```

**Comment by RoboMWM** [comment]
Forgot to make the plugin load after MV. Will fix soon.

**Comment by RoboMWM** [comment]
Fixed multiverse compat in https://github.com/RoboMWM/MidnightPortal/commit/9ff160e44fa11532c79298eff75ea3093ea2645a, the other one was unrelated and a NPE check fixed in PR #1
